Framework/module_installer.py

```diff
@@ -10,12 +10,41 @@
     import os
     DEVNULL = open(os.devnull, 'wb')
 
+from Framework.Utilities import ConfigModule
```

Comment:
In this File importing any other file is prohibited. The reason behind it is, the imported files may contain non-buitin functions which will cause errors in fresh python. So please write the ConfigModule functions inside this file instead of importing them. And please make sure you run node_cli.py with a fresh python or fresh venv to make sure installation has no error in fresh machines